src/components/questions/question.js

```diff
@@ -9,7 +9,24 @@ class Question extends React.Component{
   }
 
   render(){
-    return (<div><div>Question Content</div><AnswerList/></div>);
+    return (<div>
```

Comment:
I like the way this is set up. Any idea on how you plan on eventually styling this component and what other boilerplate you will require?

Reply:
Im still laying out that styling on paper, i will make sure to make it available once finalized!